runtime/calamari/src/lib.rs

```diff
@@ -159,7 +160,10 @@ parameter_types! {
 
 impl pallet_tx_pause::Config for Runtime {
 	type Event = Event;
-	type UpdateOrigin = EnsureRoot<AccountId>;
+	type UpdateOrigin = EnsureOneOf<
+		EnsureRoot<AccountId>,
+		pallet_collective::EnsureProportionAtLeast<_1, _2, AccountId, TechnicalCollective>,
```

Comment:
```suggestion
		pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, TechnicalCollective>,
```
Going back on my earlier opinion, please do change it

Reply:
Updated